### PR TITLE
Combined gitbook for Holepunch and Pear

### PR DESCRIPTION
**Pull request by subashsn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,120 @@
...
-* [Sharing a Pear Application](./guide/sharing-a-pear-app.md)
-* [Releasing a Pear Application](./guide/releasing-a-pear-app.md)
+## Building blocks
 
-### Reference
+The Holepunch Ecosystem is constructed utilizing the following structural components.
 
-* [Command-Line-Interface (CLI)](./reference/cli.md)
-* [Application-Programming-Interface (API)](./reference/api.md)
-* [Application Configuration](./reference/configuration.md)
```
Comment (mafintosh) [inline README.md:30]: Where did this stuff get moved to?

File: README.md [hunk 2]
```diff
@@ -1,30 +1,120 @@
...
+
+If you're already familiar with [hypercore.md](./building-blocks/hypercore.md "mention"), [hyperswarm.md](./building-blocks/hyperswarm.md "mention"), and the rest of the modules here, you might be wondering what's changed recently. In short, a lot! here's a brief breakdown of some of the highlights.
+
+### Enhanced Building Blocks
+
+Pear main focus is on making our core building blocks as easy to use, as fast, and as reliable as possible. The goal is to give developers all the essential pieces needed to develop powerful P2P apps, without being unnecessarily opinionated and without introducing operational complexity.
+
+We'd previously introduced the Hyperspace daemon and `hyp` CLI tool as core modules, but in practice found that they don't fit cleanly into this vision. We feel it's better to keep a tight focus on the fundamental building blocks so that others can build similar higher-level tools with zero friction. Given that, they are both now considered deprecated.
+
+### Hypercore v10
```
Comment (mafintosh) [inline README.md:63]: remove the versions in headers, same for below

File: README.md [hunk 3]
```diff
@@ -1,30 +1,120 @@
...
+# Pear
 
-> Pear loads applications remotely from peers and allows anyone to create and share applications with peers.
+Pear by Holepunch is a Combined Runtime, Software Development Kit (SDK), and Peer-to-Peer (P2P) Deployment tool that makes it possible to build, share, and extend P2P applications using common web and mobile technology stacks.
 
-Pear by Holepunch is a combined Peer-to-Peer (P2P) Runtime, Development & Deployment tool.
+Holepunch equips developers with a powerful suite of independent components to effortlessly construct peer-to-peer applications.
 
-Pear makes it possible to build, share and extend P2P applications using common Web and Mobile technology.
+Herein is everything needed to create unstoppable, zero-infrastructure P2P applications for Desktop, Terminal & Mobile (soon)
```
Comment (andrewosh) [inline README.md:7]: "herein is everything" -> "here you'll find everything"

**Comment by davidmarkclements** [comment]
closed for #11

**Pull request closed without merging**